etc/.golangci.yaml

```diff
@@ -93,6 +93,11 @@ issues:
     - path: /doc.go$
       linters:
         - lll
+    # TODO(RSDK-1053): reenable the staticcheck linter for this directory
+    # once deprecated API is no longer used.
```

Comment:
Bumping the version of the `api` package used in `rdk` included changes that deprecated some SLAM API. Those deprecations caused [failures](https://github.com/viamrobotics/rdk/actions/runs/4207843244/jobs/7303144377) in this PR. After talking with @nicksanford, I've disabled the `staticcheck` linter for slam services for now.